lib/tasks/db.rake

```diff
@@ -0,0 +1,16 @@
+namespace :db do
+  desc "Add constraint to votes on topics. Vote >= 0"
+  task add_constraints: :environment do
+    ActiveRecord::Base.connection.execute <<-SQL
+      ALTER TABLE topics ADD CONSTRAINT positive_votes CHECK (votes >= 0);
```

Comment:
This is the server output:
```
Started PUT "/topics/2/remove_vote?socket_id=1628.1713410" for ::1 at 2015-04-04 11:43:11 +0200
Processing by TopicsController#remove_vote as JSON
  Parameters: {"id"=>"2", "title"=>"It will all make sense...", "created_at"=>"2015-04-03T22:32:55.735Z", "updated_at"=>"2015-04-03T22:52:51.746Z", "status"=>"to_talk_about", "votes"=>0, "socket_id"=>"1628.1713410", "topic"=>{"id"=>"2", "title"=>"It will all make sense...", "created_at"=>"2015-04-03T22:32:55.735Z", "updated_at"=>"2015-04-03T22:52:51.746Z", "status"=>"to_talk_about", "votes"=>0}}
  Topic Load (0.2ms)  SELECT  "topics".* FROM "topics" WHERE "topics"."id" = $1 LIMIT 1  [["id", 2]]
   (0.2ms)  BEGIN
  SQL (0.9ms)  UPDATE "topics" SET "votes" = $1, "updated_at" = $2 WHERE "topics"."id" = $3  [["votes", -1], ["updated_at", "2015-04-04 09:43:11.741858"], ["id", 2]]
PG::CheckViolation: ERROR:  new row for relation "topics" violates check constraint "positive_votes"
DETAIL:  Failing row contains (2, It will all make sense..., 2015-04-03 22:32:55.735717, 2015-04-04 09:43:11.741858, to_talk_about, -1, null).
: UPDATE "topics" SET "votes" = $1, "updated_at" = $2 WHERE "topics"."id" = $3
   (0.1ms)  ROLLBACK
Completed 500 Internal Server Error in 5ms

PG::CheckViolation - ERROR:  new row for relation "topics" violates check constraint "positive_votes"
DETAIL:  Failing row contains (2, It will all make sense..., 2015-04-03 22:32:55.735717, 2015-04-04 09:43:11.741858, to_talk_about, -1, null).
:
  activerecord (4.2.0) lib/active_record/connection_adapters/postgresql_adapter.rb:602:in `block in exec_cache'
  activerecord (4.2.0) lib/active_record/connection_adapters/abstract_adapter.rb:466:in `block in log'
  activesupport (4.2.0) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
  activerecord (4.2 
.... etc.
```